src/shared/parse-argument.c

```diff
@@ -16,6 +18,33 @@ int parse_signal_argument(const char *s, int *ret) {
                 return 0;
         }
 
+        if (streq(s, "list")) {
+                _cleanup_(table_unrefp) Table *table = NULL;
+
+                table = table_new("signal", "name");
+                if (!table)
+                        return log_oom();
+
+                for (int i = 1; i < _NSIG; i++) {
+                        char n[STRLEN("SIGRTMIN+") + DECIMAL_STR_MAX(int)];
+
+                        xsprintf(n, "SIG%s", signal_to_string(i));
+
+                        r = table_add_many(
+                                        table,
+                                        TABLE_INT, i,
+                                        TABLE_STRING, n);
```

Comment:
use TABLE_SIGNAL for the second column. (was added recently, in a7273eaf32e9d0f8abe5f368bdd801803aaaa0ec for coredumpctl).

Reply:
Oh, that simplifies things. Done.